src/analysis/ReachingDefinitions/Ssa/PhiPlacement.h

```diff
@@ -2,6 +2,8 @@
 #ifndef _DG_PHIPLACEMENT_H_
 #define _DG_PHIPLACEMENT_H_
 
```

Comment:
This could be merged to the PhiPlacement.h commit, maybe (I don't remember how it should look at this point)